handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+# Workspaces and Role-based Access Control
+
+In SystemLink 2020R2 NI introduced a new role-based access control (RBAC) system. This security technology provides strong isolation between different *workspaces* as well as fine grain privileges for systems, data, and analysis routines in SystemLink. Collectively this capability allows access to SystemLink to scale user and data access beyond a single team to an entire organization.
```

Comment:
This feels like an odd introduction to a handbook page. This isn't a product announcement. 2020R2 was 6 months back already, and in a year or two years this won't seem like news. 

If the goal here is to call out a version requirement for the functionality described in this document, we should consider instead having a standard annotation for documents and/or sub-sections where version support may be relevant to the reader.

Reply:
Good point. I changes so it doesn't sounds like a product announcement, by removing the bit about 2020R2.